src/App.js

```diff
@@ -1,20 +1,81 @@
 import React, { Component } from 'react';
-import logo from './logo.svg';
 import Header from './Header.js';
 import ImageList from './ImageList.js';
 import imageData from './data.js';
+import SelectOptionItem from './SelectOptionItem.js';
 import './bootstrap-reboot.min.css';
 import './App.css';
 import './gallery.css';
 
 export default class App extends Component {
+  state = { selected: null };
+
```

Comment:
For your second filtered array you're checking for this.state.selectedTwo but you haven't ever initialized that state, so you'll want to add that in your state object similiar to selected.

Reply:
Really appreciate your comments, Abbey, they shoved me in the right direction!